doc/source/python/api/seldon_core.proto.rst

```diff
@@ -2,9 +2,9 @@ seldon\_core.proto package
 ==========================
 
 .. automodule:: seldon_core.proto
-    :members:
-    :undoc-members:
-    :show-inheritance:
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
Why did these change?

Reply:
Not sure, my local build of docs had to change it and it end up in the commit. I will revert that.